### DISCUSSION TITLE
Log in to MiCloud! from logs - Cannot connect to device.

### DISCUSSION THREAD

**Discussion by Myth0zz** [merdok/homebridge-miot#582] [Q&A]
Hello. I have been using the plugin for years now with no issues. I have a zhimi.heater.za2 (Smartmi 1S Heater) and a roborock.vacuum.a73 (Roborock Q8 Max) vacuum working EXCELLENT until yesterday. 

Nothing has changed in my network, but they both don't respond to the Homekit commands. In the logs they say: **Log in to MiCloud**! continuously. 

My credentials are all correct. The IP's are correct. device names - everything. Could the tokens have changed? 

I restarted router, restarted the plugin, cleared cache etc. Any help?

![Screen Shot 2024-03-06 at 12 39 38 PM](https://github.com/merdok/homebridge-miot/assets/80846065/f970b84d-0827-49e0-b15f-04442e99b7a8)

**Comment by SMarco360** [comment]
same issue... but with a fan. If you restart HB it will work again, till some point of time.

`[19.8.2024, 19:45:42] [miot] [Ventilator] Log in to MiCloud!
[19.8.2024, 19:45:47] [miot] [Ventilator] Successfully connected to MiCloud! Setting up miot device from MiCloud connection!
[19.8.2024, 19:46:49] [miot] [Ventilator] Log in to MiCloud!
[19.8.2024, 19:46:58] [miot] [Ventilator] Successfully connected to MiCloud! Setting up miot device from MiCloud connection!`

**Reply by merdok** [reply]
You would need to provide a debug log from that situation as that could potentially heave the reason for that. Without that it is impossible to tell what the reason might be...
